package.json

```diff
@@ -45,9 +45,9 @@
     "build-storybook:gatsby": "STORYBOOK_DS=core build-storybook -o packages/docs/static/storybook",
     "build-storybook:healthcare": "STORYBOOK_DS=healthcare build-storybook -o storybook-static/healthcare",
     "build-storybook:medicare": "STORYBOOK_DS=medicare build-storybook -o storybook-static/medicare",
-    "gh-pages": "yarn gh-pages:healthcare && yarn gh-pages:medicare",
```

Comment:
Removed this command that I had made a while back. It's not really a good idea to name commands the same thing as their npm bin counterpart (`./node_modules/.bin/gh-pages` was already a thing).

Reply:
I'll update the release docs